common/changes/@itwin/ecschema-editing/schema-editing-schema-difference-naming-changes_2024-06-18-09-32.json

```diff
@@ -0,0 +1,10 @@
+{
+  "changes": [
+    {
+      "packageName": "@itwin/ecschema-editing",
+      "comment": "",
```

Comment:
you made actual changes to the ecschema-editing package so you should put some comment here